src/Listeners/DeleteBreadMenuItem.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace TCG\Voyager\Listeners;
+
+use TCG\Voyager\Events\BreadDeleted;
+use TCG\Voyager\Models\MenuItem;
+
+class DeleteBreadMenuItem
+{
+    /**
+     * Create the event listener.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Delete a MenuItem for a given BREAD.
+     *
+     * @param BreadDeleted $bread
+     *
+     * @return void
+     */
+    public function handle(BreadDeleted $bread)
+    {
+        if (config('voyager.bread.add_menu_item')) {
+            $menuItem = MenuItem::where(['title' => $bread->dataType->display_name_plural]);
```

Comment:
I think its more unique/safe to select the menu-item by the route.
```suggestion
            $menuItem = MenuItem::where('route', 'voyager.'.$bread->dataType->slug.'.index');
```